src/resources/postcss/components/_small-cta.pcss

```diff
@@ -0,0 +1,31 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Small CTA (Tooltips - ET, ET+, RSVP)
+ *
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.tribe-events-c-small-cta-link {
+		@mixin desktop-body-3-bold;
+
+		border-bottom: 2px solid var(--color-border-default);
+		cursor: pointer;
+		display: inline;
```

Comment:
i'd say let's not use `inline` here, as it adds the space at the bottom for descenders we were talking about before and messes up our spacing. Instead, maybe we can use a `flex` approach on the container (`.tribe-events-c-small-cta`).